src/patients/patient-slice.ts

```diff
@@ -67,9 +67,9 @@ export const createPatient = (patient: Patient, history: any): AppThunk => async
   Toast(
     'success',
     il8n.t('Success!'),
-    `${il8n.t('patients.successfullyCreated')} ${patient.givenName} ${patient.familyName} ${
-      patient.suffix
-    }`,
+    `${il8n.t('patients.successfullyCreated')} ${patient.givenName} ${
+      patient.familyName ? patient.familyName : ''
+    } ${patient.suffix ? patient.suffix : ''}`.trimEnd(),
```

Comment:
I think rather than building the name we could use the full name property by doing `patient.fullName`

Reply:
Ah yes. Way better. Didn't realize that existed.